src/core/http-request/helpers.ts

```diff
@@ -0,0 +1,18 @@
+import { HttpMethod, HttpBody, HttpHeaders, HttpMethods } from "../http";
+import { stringBody, HttpRequestImpl } from "../../node";
+
+export type RequestParams = {
+  url: string;
+  method?: HttpMethod; // Method can also be optional in this function
+  body?: HttpBody;
+  headers?: HttpHeaders;
+};
+
+export function req({
+  url,
+  method = HttpMethods.GET,
+  body = stringBody(""),
+  headers = {}
+}: RequestParams) {
+  return new HttpRequestImpl(url, method, body, headers);
```

Comment:
We can create the string empty body if it is not provided: `const theBody = typeof body === "string" ? stringBody(body) : body`